source/manuals/monitor-paas-app-with-prometheus.html.md.erb

```diff
@@ -39,21 +39,98 @@ You can create a Prometheus service within your PaaS space and allow it to bind
 
 Within 5 minutes Prometheus will start scraping your application for metrics, you can validate this by checking [Grafana][].
 
-### App route
+### App route configuration
 
 Whether or not you're using a custom domain the Prometheus service broker will only scrape your app's first route.
 
 For example, some apps may have multiple routes especially if they use custom domains such as `custom.domain.gov.uk, an-interesting-app.cloudapps.digital`. Only the first route will be picked up.
 
-If there are no routes to your app the Prometheus service will default the route to `<app-name>.cloudapps.digital`.
+If there are no routes to your app the Prometheus service will default the route to:
+
+`<app-name>.cloudapps.digital`.
 
 Because changes to the first route will only be picked up within 5 minutes it could create gaps to metrics during route changes if you're using a zero downtime plugin such as [autopilot][].
 
+## Configure container metrics
+
+[Cloud Foundry][] provides time-series data (metrics), for your PaaS apps.
+
+Use the metrics exporter app to send Cloud Foundry metrics data to a monitoring system of your choice to store and view it. You'll need to deploy the metrics exporter as an app on PaaS before you can use it.
```

Comment:
Do we want to offer alternative solutions here? If so, we should give an example?

Reply:
I don't think we have another way of doing this at the moment

Reply:
OK, in that case, I'd reword this sentence as we're saying there are other monitoring systems people could use,  but if we don't want people to do this I'd remove this comment to avoid any confusion about what options people have and by implication is supported.